cmd/crank/beta/trace/internal/printer/default.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strings"
 
 	gcrname "github.com/google/go-containerregistry/pkg/name"
+	"github.com/liggitt/tabwriter"
```

Comment:
do we need this fork of tabwriter, or would https://pkg.go.dev/text/tabwriter work instead?